src/api/app/jobs/send_event_emails_job.rb

```diff
@@ -23,7 +23,7 @@ def perform
   def send_email(subscribers, event)
     return if subscribers.empty?
 
-    EventMailer.event(subscribers, event).deliver_now
+    EventMailer.with(subscribers: subscribers, event: event).notification_email.deliver_now
```

Comment:
According to the `EventMailer` [below](https://github.com/openSUSE/open-build-service/pull/13240/files#diff-97983eb9316b1f8a2883c0380827551d4e8a2c27edb6978860107025c9572bc0R13), if `@recipients` or `@event` is blank, `notification_email` will return `nil`. Looks like this will crash with undefined method `.deliver_now` for NilClass or something like this...

Reply:
Yeah but it's never nil....

Reply:
But then, if it's never nil, why do bother checking for `@recipients` or `@event`?

Reply:
Dunno actually 😸